### PR TITLE
Feature/update new prefab tool

### PR DESCRIPTION
**Pull request by MaxDaniels**

### CODE REVIEW

File: Editor/CreateNewPrefab.cs [hunk 1]
```diff
@@ -91,7 +125,7 @@ static bool ValidateCreatePrefabOrVariant()
     [MenuItem("FishingCactus/Tools/Create New Prefab/New or Variant")]
     static void CreatePrefabOrVariant()
     {
-        CreatePrefabs(true, false);
+        CreatePrefabs( true, false, true );
```
Comment (rascar-capac) [inline Editor/CreateNewPrefab.cs]: Can you explicitly write the parameters on those calls?

File: Editor/CreateNewPrefab.cs [hunk 2]
```diff
@@ -5,14 +5,48 @@ public class CreateNewPrefab : EditorWindow
...
     {
+        string previous_path = EditorPrefs.GetString( PrefabPathLocalKey, PrefabExtension );
+        string destination_path = string.Empty;
+
+        if( it_must_propose_path_selection )
+        {
+            destination_path = EditorUtility.OpenFolderPanel( "Destination Path :", PrefabCreationPath, "" );
+
+            if( string.IsNullOrEmpty( destination_path )
+                || AssetDatabase.IsValidFolder( destination_path )
```
Comment (rascar-capac) [inline Editor/CreateNewPrefab.cs]: shouldn’it be the negative instead? A valid folder should result in false

**Review by rascar-capac — CHANGES_REQUESTED** [review]

File: Editor/CreateNewPrefab.cs (hunk 2 above)
Comment (MaxDaniels) [reply Editor/CreateNewPrefab.cs]: Done.

File: Editor/CreateNewPrefab.cs (hunk 1 above)
Comment (rascar-capac) [reply Editor/CreateNewPrefab.cs]: Maybe you missed this comment?

Comment (MaxDaniels) [reply Editor/CreateNewPrefab.cs]: done.

**Review by rascar-capac — APPROVED** [review]

**Pull request merged**